src/main/kotlin/mate/academy/Main.kt

```diff
@@ -1,7 +1,20 @@
 package mate.academy
 
+const val GREETING_TEXT = "Please enter a name (or press Enter to quit):"
+
 fun greetUsers() {
-    // implement function
+    fun greetingText() : String = GREETING_TEXT
```

Comment:
This method has low functionality due to just returning the const val. You may use GREETING_TEXT directly